auslib/test/blobs/test_apprelease.py

```diff
@@ -1629,6 +1629,123 @@ def testSchema5OptionalAttributes(self):
         self.assertEqual(returned_footer.strip(), expected_footer.strip())
 
 
+class TestSchema6Blob(unittest.TestCase):
```

Comment:
Since the only change between schema 5 and schema 6 is the removal of attributes, it would be good to have a test that ensures a schema 6 blob fails validation if one of those fields is present.